{{cookiecutter.project_shortname}}/INSTALL.rst

```diff
@@ -34,6 +34,39 @@ Elasticsearch {{cookiecutter.elasticsearch}}, RabbitMQ and Redis):
 Next, bootstrap the instance (this will install all Python dependencies and
 build all static assets):
 
+Creating an environment
+-----------------------
+
+In order to bootstrap and install your instance, it is recommended to use a
+virtual environment. For that we will use ``pipenv``.
+
+.. code-block:: console
+
+    $ pipenv shell
```

Comment:
The bootstrap script will also create the environment. The intention with bullet 6 in #102 was just to document how you enter the virtual environment once it has already been created, so I would move it below in-between the "Upgradeing" and "Testing" section. Also, I wouldn't show output as we're not showing any output from the other commands in the doc.